backend/courses/views.py

```diff
@@ -171,9 +200,11 @@ class RequirementList(generics.ListAPIView, BaseCourseMixin):
     schema = PcxAutoSchema(
         examples=examples.RequirementList_examples,
         response_codes={
-            "/api/plan/requirements/": {"GET": {200: "[SCHEMA]Requirements listed successfully."}},
-            "/api/courses/{semester}/requirements/": {
-                "GET": {200: "[SCHEMA]Requirements listed successfully."}
+            "/api/plan/requirements/": {
+                "GET": {200: "[DESCRIBE_RESPONSE_SCHEMA]Requirements listed successfully."}
+            },
+            "/api/base/{semester}/requirements/": {
```

Comment:
This should probably use the `reverse()` function in django instead of hard-coding in URLs -- sorry for not catching this earlier, I know this isn't strictly new for this PR. I think as a general rule, URLs should always only show up in the URL file so we don't have to do big find-and-replaces if we ever change URLs (like now, haha)